.github/workflows/document.yaml

```diff
@@ -0,0 +1,52 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    paths: ["R/**", "README.Rmd"]
+  workflow_dispatch:
+
+name: Document
+
+jobs:
+  document:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      DELPHI_EPIDATA_KEY: ${{ secrets.DELPHI_GITHUB_ACTIONS_EPIDATA_API_KEY }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: |
+            any::devtools
+            any::roxygen2
+          needs: |
+            devtools
+            roxygen2
+
+      - name: Document
+        run: roxygen2::roxygenise()
+        shell: Rscript {0}
+
+      - name: Build README.md from README.Rmd
+        run: Rscript -e 'devtools::build_readme()'
+
+      - name: Commit and push changes
+        run: |
+          git config --local user.name "$GITHUB_ACTOR"
+          git config --local user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          git add README.md
+          git add man/\* NAMESPACE DESCRIPTION
+          git commit -m "docs: document (GHA)" || echo "No changes to commit"
+          git pull --rebase
```

Comment:
issue: it might be possible to get into bad race conditions here with document + style on examples.  If you think Git will catch this and balk, then maybe it's okay this way.  But this ff-only + [this concurrency group thing](https://docs.github.com/en/actions/using-jobs/using-concurrency#example-concurrency-groups) might work better if it does what I think.

Reply:
Hm, given that we've been exposed to this potential race condition in epidatr for a while now and haven't had an issue, I figure Git is taking care of it. Can find a fix if we get a mysterious error here.

Reply:
So there may be some more low-probability races between the final push&pulls; I think Git can detect those.  But more commonly, I imagine you could have:
- doc & style start simultaneously
- styler finishes, pulls, pushes
- document finishes, pulls & rebases, pushes
- document still needs to run another time because styler updated example source.  Git does not detect conflict unless styler also re-documents after touching examples.  What does workflow_dispatch: do?  (And if it saves us here, how does it not get into infinite loops?)

Reply:
I guess concurrency groups probably aren't the right thing either.  Just need an order: style first, then document.  Not sure about lintr thing, maybe that's just a check that could be in parallel.

Reply:
workflow_dispatch lets you queue another document command that this time contains the styling changes, if you wish. In practice, the next commit you push will run one or the other again and that will get you back to a stable state, so it hasn't been an issue.

Reply:
Oh, I guess maybe document will be triggered to re-run if styler does update examples.  Okay, maybe there's no uncaught bad stuff that I can think of happening.

Reply:
Yeah, if there is that super rare chance of the final pull-pushes causing Git to balk and error out one of the actions, we can just dispatch it.